gatsby-node.js

```diff
@@ -111,51 +171,26 @@ const createLiveGhostNodes = ({actions}, configOptions) => {
             }
         }
 
-        setting.codeinjection_styles = _.isNil(setting.codeinjection_styles) ? '' : setting.codeinjection_styles;
```

Comment:
This will result in possible breaking change - previously this field would result in empty string (if `_.isNil` check is truthy), with removal this will result in `null`. This can cause problems if someone tries to some string manipulation assuming that this field is never null

Reply:
In reference to above, this does not match the return pattern of `transformCodeinjection()`. This would be a breaking change. Should this be normalised to be entirely nullable or entirely non-nullable?